### ISSUE TITLE
Should the following sentence be in a cell?

### ISSUE THREAD

**Issue by tom-osika** [johnwparent/APBPoster#24]
https://github.com/johnwparent/APBPoster/blame/master/wrapper_yaml.rst#L104

**Comment by johnwparent** [comment]
No, that's where I intended it to be. This is the same as the APB readme. Do you think it should be moved elsewhere?

**Comment by tom-osika** [comment]
No looking at it again its fine. I might just restructure the sentence a bit.
